manifests/userlist.pp

```diff
@@ -13,4 +14,11 @@
     content => template('pgbouncer/userlist.txt.erb'),
     order   => '01',
   }
+
+  concat::fragment { "${paramtmpfile}_users":
```

Comment:
I missed this in my local testing prior to merge. This works if the userlist is only called once, but is duplicated and causes compilation issues if it is called a second time. I'm going to re-work this (and the supporting files) to allow multiple calls to the defined type. My initial thoughts are tying in the username like what happens above, and then changing up the template to have an additional piece so it doesn't re-declare the `[users]` section of the config. I wanted to give you a heads up that this will be changing so you know to pull and perform any re-tests as necessary with the new release. I'm working through the fix today and will post the new release as soon as possible (should be this week).